custom_components/frigate/media_source.py

```diff
@@ -461,13 +463,27 @@ def get_integration_proxy_path(self) -> str:
             and self.hour is not None
         ):
             year, month, day = self.year_month_day.split("-")
+            tz_offset = int(
+                (
+                    dt.datetime.now(
+                        pytz.timezone(timezone.replace(",", "/"))
```

Comment:
Why is this replace necessary? Since we're not using timezones in paths anymore, it would be better it would could use the standard timezone specifiers (with the slashes).

Reply:
I think that was just a copy pasta error